pkg/reconciler/pipelinerun/resources/resultrefresolution.go

```diff
@@ -71,29 +71,6 @@ func validateArrayResultsIndex(allResolvedResultRefs ResolvedResultRefs) (Resolv
 	return allResolvedResultRefs, "", nil
 }
 
-// extractResultRefs resolves any ResultReference that are found in param or pipeline result
-// Returns nil if none are found
-func extractResultRefsForParam(pipelineRunState PipelineRunState, param v1beta1.Param) (ResolvedResultRefs, error) {
```

Comment:
These two functions are currently not being used anywhere, so they can be cleaned up.